doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+this matter.
+
+To understand the foundation for the Data Warehouse, let's start with the `long
+term vision` that guided our work.
+
+##  Long term vision
+
+```
+Give departments the data they need, at the point they need it so that they can
+manage their GOV.UK content at scale over time.
+```
+
+Let's explore in detail:
+
+## Give departments the data they need
+
+Departments manage content daily. They publish, update and archive content as
+part of their daily work; as of today, content designers don't have a single
+place to get metrics about how their content is performing. This information is
+beneficial to make informed decisions about their next steps.
+
+As per user research, content designers would benefit from having two types of
+metrics.
+
+* Metrics about **content performance**:
+
+```
+Metrics that define how content behaves when it interacts with GOV.UK users. For
+example, the number of visits, the number of searches within the page, the
+number of feedback comments that a page is receiving, or the overall
+satisfaction with a particular page.
+```
+
+These metrics are tracked daily.
+
+* Metrics about **content quality**:
+
+These metrics are tracked/updated on each new version.
+
+```
+Metrics that define core characteristics of content: the reading score, use of
+passive voice, guidelines compliance, number of paragraphs, words, complex sentences,...
+```
+
+## At scale and over time
+
+`At scale` and `over time` key principles for the Data Warehouse, because
+content designers are interested in:
+
+1. Metrics for each content item that is or have been live in GOV.UK (around
+   550K)
+2. Trends of each of the metrics **over time**
+
+To explain in detail why **over time** is important, let us explore three simple
+examples.
+
+### Example 1: Number of Content Items published by an organization
+
+Departments are content owners; this means that they are responsible for
+publishing, updating and archiving/retiring their content daily. Diligent
+management of content implies a steady pace of updates and publications while
+keeping the total number of content items in a sensible number.
+
+For example, if an organization is publishing at a very high rate for an
+extended period of time, it could be a signal that they need to improve the way
+they archive or retire existing content.
+
+To get these insights about content publication we can't focus on metrics for a
+single day, we need to understand how those metrics evolved through time; we
+need to explore metrics **over time**.
+
+Let's explore the example:
+
+* **Content Items published by an organization in one day**
+
+We have two organizations: `A` and `B`. In the last day, `Organisation A` has
+`550` live Content Items and `Organisation B` has `400`.
+
+With this information, it is not easy to say if any of those organization are
+not doing good management of their content because we only have a metric on a
+particular day of the year.
```

Comment:
With this information, it is hard to see if those organizations are effectively managing their content because we only have a data points for a single day.